src/Rules/IRuleMetadataProvider.cs

```diff
@@ -40,10 +41,32 @@ public class RuleMetadataProvider : IRuleMetadataProvider
     {
         public IRuleHelp GetRuleHelp(Language language, string ruleKey)
```

Comment:
I'm thinking about renaming this method to `GetRuleDescription` and `IRuleHelp` to `IRuleDescription` in a separate PR - seems a bit more accurate than `Help`. Let's discuss this next week.